.gitignore

```diff
@@ -7,6 +7,8 @@
 # Ignore bundler config.
 /.bundle
 
+/config/initializers/app_keys.rb
```

Comment:
This seems to be repeated in another PR. Make sure to rebase once of the PR is merged.